src/Parser.js

```diff
@@ -1,3 +1,4 @@
+const path = require('path')
```

Comment:
markedpp runs also in the browser. 
Can you please move the intended functionality regarding the vscode link into ppInclude.js and ppInclude-browser.js.
Requiring path here will cause that the browser package can't be processed with webpack any longer.

Reply:
Ok, moved.